tests/src/python/utilities.py

```diff
@@ -816,7 +816,7 @@ def memberIsDocumented(self, member_elem):
 def waitServer(url, timeout=10):
     """ Wait for a server to be online and to respond
         HTTP errors are ignored
-        @param timeout: in seconds
+        \param timeout: in seconds
```

Comment:
Watch out here, that should probably be `:param timeout:`? (Python specialists required ;) )